api/src/pcapi/routes/auth/forms/empty.py

```diff
@@ -0,0 +1,5 @@
+from flask_wtf import FlaskForm
+
+
+class EmptyForm(FlaskForm):
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Tu as encore besoin de ça ?‏